A few of gorry's nits (#4085)
quicwg/base-drafts: draft-ietf-quic-recovery.md

Some networks may exhibit higher degrees of packet reordering, causing a sender
to detect spurious losses. Additionally, packet reordering could be more common
with QUIC than TCP, because network elements that could observe and reorder

> out-of-order TCP packets cannot do that for QUIC, because packet numbers

Reviewer: ```suggestion
TCP packets cannot do that for QUIC, because packet numbers
```

a timeout in the Initial packet number space doubles the length of the timeout
in the Handshake packet number space.

> The total length of time over which consecutive PTOs expire are limited by the

Reviewer 2: ```suggestion
The total length of time over which consecutive PTOs expire is limited by the
```

> The time length of a connection that is experiencing consecutive PTOs is
> limited by the endpoint's idle timeout.

Reviewer: ```suggestion
The total length of time over which consecutive probes are sent is limited by the idle timeout.
```
?

Author: Sure, works for me.

Author: I'm honestly not sure we need this sentence, but I suspect it was added because someone asked for it?
